src/engine/LoadingScreen.cs

```diff
@@ -31,11 +33,17 @@ public class LoadingScreen : Control
     private Label tipLabel;
     private Control spinner;
 
-    private string loadingMessage = TranslationServer.Translate("LOADING");
+    private bool visible;
+
+    private Random random;
+    private Timer randomizeTipTimer;
+
+    private string loadingMessage;
+    private string tip;
     private string loadingDescription = string.Empty;
-    private string tip = TranslationServer.Translate("LOADING_TIP");
 
     private float totalElapsed;
+    private MainGameState target = MainGameState.Invalid;
```

Comment:
this should be named "currentlyLoadingGameState" as this loading screen doesn't "target" anything in my opinion.